### PR TITLE
EPMRPP-77247 || Add information to "Terms & Conditions" section

### PR DESCRIPTION
**Pull request by Vadim73i**

### CODE REVIEW

File: src/react-components/pages/terms-page/termsPage.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import reactWrapper from 'utils/reactWrapper';
+import ReactMarkdown from 'react-markdown'
+import BodyLayout from 'react-components/layouts/bodyLayout.jsx';
+import Button from 'react-components/common/button/button.jsx';
+import termsMarkdownData from 'react-components/pages/terms-page/termsMarkdownData';
+import styles from './termsPage.scss';
+
+const cx = classNames.bind(styles);
+
+const Layout = () => (
```
Comment (AmsterGet) [inline src/react-components/pages/terms-page/termsPage.jsx]: Why Layout? Why not `TermsPage`?

**Review by chivekrodis — APPROVED** [review]

File: src/views/components/footer/Footer.jade [hunk 2]
```diff
@@ -20,3 +20,5 @@
       a.privacy(href='https://privacy.epam.com/core/interaction/showpolicy?type=CommonPrivacyPolicy' target='_blank') Privacy Policy
       .separator
       a.cookie(href='https://privacy.epam.com/core/interaction/showpolicy?type=CommonCookiePolicy' target='_blank') Cookie Policy
+      .separator
+      a.terms(href='/legal/terms' target='_blank') Terms & Conditions
```
Comment (AmsterGet) [inline src/views/components/footer/Footer.jade:24]: I see two different separators in Chrome on Windows (see the screenshot below).

image

File: src/react-components/pages/terms-page/termsPage.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  <BodyLayout className={cx('terms-page-wrapper')}>
+    <div className={cx('terms-page')}>
+      <div className={cx('title')}>Terms and Conditions</div>
+      <div className={cx('content')}>
+        <div className={cx('description')}>The following Terms and Conditions (“Terms”) govern Customer’s use of the Report Portal Platform and any Services provided by Test IO, Inc (“Test IO”).  The Report Portal Platform and any other software is licensed and not sold.</div>
+        <a href='/downloads/ReportPortal%20Terms%20of%20Service%20-%20PDF.pdf' download>
```
Comment (AmsterGet) [inline src/react-components/pages/terms-page/termsPage.jsx]: These area should not be clickable (see the screenshot below).

image

File: src/react-components/pages/terms-page/termsMarkdownData.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  **14.7.\tWaivers.**  A waiver of any default hereunder or of any of the terms and conditions of this Agreement will not be deemed to be a continuing waiver or a waiver of any other default or of any other term or condition, but will apply solely to the instance to which such waiver is directed. The exercise of any right or remedy provided in this Agreement will be without prejudice to the right to exercise any other right or remedy provided by law or equity, except as expressly limited by this Agreement.\\
+  **14.8.\tEntire Agreement.**  This Agreement, including any attachments or exhibits constitutes the entire agreement between the parties. All prior negotiations, proposals, and agreements between the parties concerning the subject matter contained in this Agreement, are canceled and superseded by this Agreement. Any changes to this Agreement must be agreed to by both parties in writing.
+`;
+
+export default termsMarkdownData;
```
Comment (AmsterGet) [inline src/react-components/pages/terms-page/termsMarkdownData.js]: I guess it should be a `.md` file and it will be read as string from JS.

**Review by AmsterGet — CHANGES_REQUESTED** [review]
Please remove _- PDF_ part in the file name.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**